docs/usage.rst

```diff
@@ -8,8 +8,9 @@ For these examples, let's assume the current folder is a copier template. it sho
 .. code-block::
 
    my_template/
-   ├── {{repo_name}}/
-   │   └── README.rst.jinja
+   ├── template
+   ├────── {{repo_name}}/
```

Comment:
I was initially thinking that a template folder could be templated (like in cookiecutter) but it's not. So I agree with to set a "template" folder instead. 
On the other hand, I think `{{repo_name}}` should be removed from the examples.